<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyNote</class>
 <widget class="QMainWindow" name="MyNote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>405</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>405</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>405</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MyNote</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sources.qrc">
    <normaloff>:/img/images/note.png</normaloff>:/img/images/note.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>560</width>
      <height>360</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_calendar" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>560</width>
       <height>360</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>560</width>
       <height>360</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>560</width>
       <height>360</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_Saturday">
      <property name="geometry">
       <rect>
        <x>485</x>
        <y>85</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 124, 52);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SAT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_Friday">
      <property name="geometry">
       <rect>
        <x>435</x>
        <y>85</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 214, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>FRI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_Thursday">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>85</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 214, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>THU</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_Wednesday">
      <property name="geometry">
       <rect>
        <x>335</x>
        <y>85</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 214, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>WED</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_Tuesday">
      <property name="geometry">
       <rect>
        <x>285</x>
        <y>85</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 214, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TUE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_Monday">
      <property name="geometry">
       <rect>
        <x>235</x>
        <y>85</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 214, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>MON</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_Sunday">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>85</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 124, 52);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SUN</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_day_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>55</width>
        <height>110</height>
       </rect>
      </property>
      <property name="pixmap">
       <pixmap>:/src/Images/Number/0_yellow.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_day_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>55</width>
        <height>110</height>
       </rect>
      </property>
      <property name="pixmap">
       <pixmap>:/src/Images/Number/8_yellow.png</pixmap>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNewSchedule"/>
    <addaction name="actionManageSchedule"/>
    <addaction name="actionSetting"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewSchedule">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/img/images/new.png</normaloff>:/img/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建日程</string>
   </property>
   <property name="toolTip">
    <string>新建日程</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/img/images/logout.png</normaloff>:/img/images/logout.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/img/images/settings.png</normaloff>:/img/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionManageSchedule">
   <property name="text">
    <string>查看日程</string>
   </property>
   <property name="toolTip">
    <string>查看日程</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections/>
</ui>
